ocs_ci/deployment/helpers/ceph_cluster.py

```
@@ -482,3 +483,224 @@ def simulate_full_ceph_bluestore_process_on_wnodes(
+
+    Args:
+        wnodes (list): List of worker node objects where the simulation
+            should be performed.
+        namespace (str): Namespace for the debug pod.
+
+    Returns:
+        bool: True if the simulation succeeded on all nodes, False otherwise.
+
+    """
+    for wnode in wnodes:
+        logger.info(f"Simulating encrypted Ceph BlueStore on worker node: {wnode.name}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)